fe/fe-core/src/main/java/com/starrocks/analysis/TypeDef.java

```diff
@@ -92,6 +94,19 @@ public void analyze() throws AnalysisException {
         isAnalyzed = true;
     }
 
+    public void analyze(boolean isOlap) throws AnalysisException {
+        if (isOlap && (!FeConstants.runningUnitTest)) {
+            // we haven't support create table with map or struct type column in native table
+            Type innerType = Type.getInnermostType(parsedType);
+            if (innerType.isMapType() || innerType.isStructType()) {
+                throw new AnalysisException("Unsupported data type: " + parsedType.toSql());
+            }
+            analyze();
+        } else {
```

Comment:
put analyze() to outside, there is no need to write else branch.

Reply:
done